src/icons/icon_component_instance.pr

```diff
@@ -0,0 +1,6 @@
+{[ let configuration = context /]}
+
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7 1.5L1.5 7L7 12.5L12.5 7L7 1.5Z" stroke="#606981" stroke-width="1.1" stroke-linecap="square"/>
```

Comment:
Check other icons. Stroke should not be hardcoded but rather used with currentColor

Reply:
[b15bba4](https://github.com/Supernova-Studio/exporter-documentation/pull/166/commits/b15bba451bb361bb9f9331407f89d98a736169f4)